### ISSUE TITLE
Tensorflow 1.2.0 Win64, GPU and AVX enabled

### ISSUE THREAD

**Issue by aluo-x** [yaroslavvb/tensorflow-community-wheels#24]
More details are in the readme.
https://github.com/aluo-x/tensorflow_windows

**Comment by bconstanzo** [comment]
This would be great for Python 3.6!

**Comment by aluo-x** [comment]
I have discovered that batchnorm is broken is the current build. I would advise against using my build until further investigation. A discussion can be found in my repo.

**Comment by bconstanzo** [comment]
Thanks for the warning, though for the moment I'm using Crhistoph Gohlke's build (Windows, Py3.6, no SSE, no nothing, but works).

**Comment by bconstanzo** [comment]
I see in the repo that you've been working on this. Here's hoping that you figure the broken things on the 3.6 builds -- I'll test as soon as I you upload, and thanks for the hard work!

**Comment by aluo-x** [comment]
So I filed an issue here [#11865](https://github.com/tensorflow/tensorflow/issues/11865), awaiting a response.

**Comment by aluo-x** [comment]
I've added a CPU only version w/ AVX2, built using VS 2017 15.4 and Python 3.6.3. 
So far GPU versions with CUDA 9 and cuDNN 7 are still failing on Windows, but will likely be added with TF 1.5.

**Comment by aluo-x** [comment]
just uploaded a whl with CUDA 8 + cudnn 7 + AVX (not 2) support.

**Comment by bconstanzo** [comment]
You sir, are a gentleman. I'll be checking (and most probably using) those AVX builds!
